--- ./tinyusb/device/usbd.c	2017-11-02 07:14:29.862559010 -0700
+++ ./tinyusb/device/usbd.c	2017-11-02 07:16:07.963064778 -0700
@@ -308,6 +308,7 @@
 // may need to open interface before set configured
 static tusb_error_t usbd_set_configure_received(uint8_t coreid, uint8_t config_number)
 {
+  (void)config_number;
   dcd_controller_set_configuration(coreid);
   usbd_devices[coreid].state = TUSB_DEVICE_STATE_CONFIGURED;
 
@@ -349,6 +350,7 @@
 
 static tusb_error_t get_descriptor(uint8_t coreid, tusb_control_request_t const * const p_request, uint8_t const ** pp_buffer, uint16_t * p_length)
 {
+  (void)coreid;
   tusb_std_descriptor_type_t const desc_type = (tusb_std_descriptor_type_t) u16_high_u8(p_request->wValue);
   uint8_t const desc_index = u16_low_u8( p_request->wValue );
 
